docs/user_manual/calculations.md

```diff
@@ -82,11 +85,15 @@ The number of measurements can be found by taking the sum of the following:
 - two times the number of nodes with a voltage sensor with magnitude and angle
 - two times the number of nodes without appliances connected
 - two times the number of nodes where all connected appliances are measured by a power sensor
-- two times the number of branches with a power sensor
+- two times the number of branches with a power sensor and/or a current sensor
 
 ```{note}
 Having enough measurements does not necessarily mean that the system is observable. The location of the measurements is
-also of importance. Additionally, there should be at least one voltage measurement.
+also of importance, i.e., the measurements should be topologically independent. Additionally, there should be at least one voltage measurement.
+```
+
+```{note}
+Global angle current measurements require at least one voltage angle measurement to make sense.
 ```
```

Comment:
Cany you move this section into the warning mentioned below:
"For this reason, global angle current measurements require at least one voltage angle measurement."

Reply:
incorporated cfr. offline discussion